revenuecat_examples/purchase_tester/lib/src/constant.dart

```diff
@@ -6,3 +6,5 @@ const googleApiKey = 'googl_api_key';
 
 //TO DO: add the Amazon API key for your app from the RevenueCat dashboard: https://app.revenuecat.com
 const amazonApiKey = 'amazon_api_key';
+
+const entitlementKey = 'pro';
```

Comment:
This was `pro_cat` and now is `pro`. Is that ok?

Reply:
feels like just fixing the entitlement thing is a very minor refactor? i.e.: just change wherever we look for `entitlements[entitlementKey]` and replace it for `entitlements.first`? Not sure we need a whole rewrite for that

Reply:
Here's ChatGPT's 2 min-take on this: 

<img width="912" alt="Screenshot 2023-04-03 at 4 44 49 PM" src="https://user-images.githubusercontent.com/3922667/229611417-304782be-c79f-4eed-b93c-0930f492969d.png">
https://github.com/RevenueCat/purchases-flutter/commit/7bf53e937e8c314ea504775f6ffaa8dfe82090dd

Reply:
@aboedo I didn't mean rewritten and I didn't _just_ for the entitlement key 😇 The purchase tester that Android and React Native have where it can list all packages and then offer all ways to purchase (as package, as product, and as subscription option) is what I was really referring to about the rewrite.

Like this 👇 Testing these changes in React Native has been a breeze and I really want this for Flutter too 💪 

https://user-images.githubusercontent.com/401294/230485791-0afe36bf-670d-48b9-b1c9-77b185f03e5a.mov